engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -489,10 +491,13 @@ private void updateCamera(CharacterMovementComponent charMovementComp, Vector3f
             }
         }
 
-        if (charMovementComp.mode == MovementMode.GHOSTING) {
-            playerCamera.extendFov(24);
-        } else {
-            playerCamera.resetFov();
+        if(charMovementComp.mode != lastMode){
+            if (charMovementComp.mode == MovementMode.GHOSTING) {
```

Comment:
`noclip` sounds like a very technical term that only people familiar with the opengl meaning of clipping might understand. 

`spectator` is as good as `ghosting`: I'd still be wondering what it means in practical terms in the context of movement modes. I feel the ghosting term can stay. I'd just add the comment above to the MovementMode file.